src/conditionals/simple_programs_with_decision_points/medium/q1/InvestOpTest.java

```diff
@@ -0,0 +1,29 @@
+package conditionals.simple_programs_with_decision_points.medium.q1;
+
+/*
+Write a program that will ask the user for their investment and double it if it is above $5000
+ */
+
+import global.BaseTest;
+import global.variables.Clause;
+import global.variables.clauses.IntegerLiteral;
+import global.variables.clauses.NewLine;
+import global.variables.clauses.RandomInteger;
+import global.variables.clauses.StringLiteral;
+
+public class InvestOpTest extends BaseTest {
+
+    public Clause[] testSentence() {
+        return new Clause[]{
+                new StringLiteral("What is your investment in $ (rounded to the nearest dollar)? "),
```

Comment:
It might be escaping your special characters (`$`, `(`, `)`) with `\\`. So `"What is your investment in \\$..."`

Reply:
Ok thanks, I thought the $ sign might cause problems, but I thought the brackets would be fine

Reply:
Yeah almost every non-alphanumeric is special so best to just escape sometimes - hopefully this helps!